src/pages/patientView/mutation/column/AlleleFreqColumnFormatter.tsx

```diff
@@ -31,12 +32,30 @@ export default class AlleleFreqColumnFormatter {
 
             const bar = (<rect x={barX[mutation.sampleId]} y={barY} width={AlleleFreqColumnFormatter.barWidth} height={barHeight} fill={sampleManager.getColorForSample(mutation.sampleId)}/>);
 
-            const circleRadius = 6;
             const sampleId = mutation.sampleId;
-            const component = sampleManager.getComponentForSample(sampleId);
+            const isUncalled = mutation.geneticProfileId.endsWith(GENETIC_PROFILE_UNCALLED_MUTATIONS_SUFFIX);
```

Comment:
I don't think endsWith is supported in IE or safari < 9.    i would build a regexp to do this.  
new RegExp(`${GENETIC_PROFILE_UNCALLED_MUTATIONS_SUFFIX}$`);